intoto.py

```diff
@@ -660,13 +662,17 @@ def _intoto_verify(message_data):
           " installation continues.")
 
     else:
-      # Notify apt about the failure ...
+      # Notify apt about the failure using a short error message
+      error_msg = ("In-toto verification failed with '{}'.".format(
+          type(e).__name__))
       notify_apt(URI_FAILURE, error_msg, uri)
       # ... and do not relay http's URI Done (so that apt does not install it)
       return False
 
   else:
-    logger.info("In-toto verification for '{}' passed! :)".format(filename))
+    # Colorize (blue) success message
+    logger.info("\033[34mIn-toto verification for '{}' passed! :)\033[0m"
+        .format(filename))
```

Comment:
The SSL guidelines seem to favor string concatenation over string formatting. Not sure if this is a big deal.

Reply:
in-toto projects favors `format` over string concatenation pretty much everywhere. :)